dbos-config.schema.json

```diff
@@ -145,6 +145,13 @@
         "admin_port": {
             "type": "number",
             "description": "The port number of the admin server (Default: 3001 or the app server port + 1)"
+        },
```

Comment:
Just to confirm, this is exactly the same as in Python?

Reply:
The start command? Yes, I checked the Python implementation for reference. It
- Takes a list of commands
- Run each of them (blocking)